lib/personalAccessKey.ts

```diff
@@ -161,22 +179,16 @@ export async function accessTokenForPersonalAccessKey(
 export async function enabledFeaturesForPersonalAccessKey(
   accountId: number
 ): Promise<{ [key: string]: number } | undefined> {
-  const account = getAccountConfig(accountId) as PersonalAccessKeyAccount;
-  if (!account) {
-    throwErrorWithMessage(`${i18nKey}.errors.accountNotFound`, { accountId });
-  }
-  const { auth, personalAccessKey, env } = account;
-  const authTokenInfo = auth && auth.tokenInfo;
-
-  const accessTokenResponse = await getNewAccessToken(
-    accountId,
-    personalAccessKey,
-    authTokenInfo && authTokenInfo.expiresAt,
-    env
-  );
+  const accessTokenResponse = await getNewAccessTokenByAccountId(accountId);
```

Comment:
Rather than exposing one-off utils to access fields on the access token response, I think we might be better off just exposing a util to get the access token for a given account id. Then we won't have to come in here and expose new utils every time we want to access a new field on the access token response. 

We'll probably have to keep `enabledFeaturesForPersonalAccessKey` around though, just to prevent this form becoming a breaking change.